library/build.gradle

```diff
@@ -21,7 +21,7 @@ apply plugin: 'realm-android'
 apply plugin: 'com.jfrog.bintray'
 apply plugin: 'maven-publish'
 
-version '0.6.1'
+version '0.6.2'
```

Comment:
I don't think that this should be here? Or rather should have been changed in this PR

Reply:
Not sure what the suggestiion is here. 

When/where should this be done/placed?

Reply:
Revert it back to the previous version

Reply:
I need it to be a higher version?  

I want to publish it. What is you're suggestion?

Reply:
Kindly create a different PR when making a release when you are done with the feature(s) required for that release and merged to master

Reply:
Cool thanks.